README.md

```diff
@@ -1 +1,90 @@
-JavaScript-Capstone-Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Mohammed Salah Personal Portfolio
+
+This is a web site for the IT Live Summit for 2019.
+
+# Who is IT.LIVE
+
+It is the first Iraqi conference concerned with raising various issues
+related to information technology, as this field
+has not been addressed across the country through previous conferences and activities, and this is
+what our conference
+aims to achieve.
+What can we look forward to in #IT_Live_2017?
+There will be a number of experts in this field discussing different topics, a number of
+competitions, entertainment
+events and gifts, companies and individuals with experience in this field who will present their
+ideas and other
+activities that will be announced soon.
+When is the conference date?
+End of February 2017, dates and times will be announced soon along with the application form for
+attendees and speakers.
+
+# Screen Shots
+
+## Desktop Version
+
+![screenshot](./images/screeshots/desktop-home.png)
+
+![screenshot](./images/screeshots/desktop-about.png)
+
+## Mobile Version
+
+![screenshot](./images/screeshots/mobile-home.png)
+
+![screenshot](./images/screeshots/mobile-about.png)
+
+## Built With &hearts;
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Ckeckout The live Site here](https://mo7amedsmahdi.github.io/JavaScript-Capstone-Project/)
```

Comment:
- [ ] The demo link still shows the old version of your website. kindly deploy it again with the new branch so your demo link shows your current work.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/77190203/158038326-312f9a6c-a5c4-4179-9f1b-94624b587ae6.png">